report/renderer/crc-details-renderer.js

```diff
@@ -106,33 +106,33 @@ class CriticalRequestChainRenderer {
     // Construct lines and add spacers for sub requests.
     segment.treeMarkers.forEach(separator => {
       if (separator) {
-        treeMarkeEl.appendChild(dom.createElement('span', 'lh-tree-marker lh-vert'));
-        treeMarkeEl.appendChild(dom.createElement('span', 'lh-tree-marker'));
+        treeMarkeEl.append(dom.createElement('span', 'lh-tree-marker lh-vert'));
```

Comment:
oh man this little sequence is funny. 
dunno about others but i think doing some ternaries to tweak the classname would make a lot of sense rather than these if/else blocks

might as well refactor this here as we never touch this file

Reply:
Refactored this a bit; resolving.